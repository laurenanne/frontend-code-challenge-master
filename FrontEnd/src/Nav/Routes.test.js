import React from "react";
import { render } from "@testing-library/react";
import AppRoutes from "./Routes";
import { MemoryRouter } from "react-router";

it("renders without crashing", function() {
  render(
    <MemoryRouter>
      <AppRoutes />
    </MemoryRouter>
  );
});

it("matches snapshot", function() {
  const { asFragment } = render(
    <MemoryRouter>
      <AppRoutes />
    </MemoryRouter>
  );
  expect(asFragment()).toMatchSnapshot();
});
