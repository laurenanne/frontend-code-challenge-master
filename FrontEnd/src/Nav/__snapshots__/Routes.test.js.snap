// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="routes"
  >
    <div
      class="home mt-5 container text-center vh-100"
    >
      <h1>
        Yodlr Design Challenge
      </h1>
      <div
        class="m-3"
      >
        <a
          href="/signup"
        >
          <button
            class="btn btn-outline btn-light btn btn-secondary"
            style="border-color: #8f5cd1; color: rgb(143, 92, 209);"
          >
            Registration Page
          </button>
        </a>
      </div>
      <div
        class="m-3"
      >
        <a
          href="/admin"
        >
          <button
            class="btn btn-outline btn-light btn btn-secondary"
            style="border-color: #8f5cd1; color: rgb(143, 92, 209);"
          >
            Admin Page
          </button>
        </a>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
